### ISSUE TITLE
Well done on having local database set  up instructions!!👌😍

### ISSUE THREAD

**Issue by rooberrydev** [fac19/week6-CADO#51]
DATABASE_URL probably comes from your heroku database btw, heroku usually sets this up as a connection string of the format 

```postgres://YourUserName:YourPassword@YourHost:5432/YourDatabase```

if you had a DATABASE_URL environment variable locally with the connection string in that format, then it works as expected (overrules the other environment variables)

but I understand that locally you're using env variables for each field (username, password, db name) which means having a database url env on top of that would be redundant.

https://github.com/fac19/week6-CADO/blob/e467a594478483a2a5e3c4471118b2c5cf690e37/source/database/connection.js#L6-L10

TL:DR having DATABASE_URL locally shouldnt error if the connection string is in the right format
